docs/events.md

```diff
@@ -6,11 +6,12 @@ weight: 700
 -->
 # Events in Tekton
 
-Tekton's task controller emits [Kubernetes events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#event-v1-core)
```

Comment:
just some grammar nits: `controllers emits` -> `controllers emit` and `the underlying TaskRun do` -> `the underlying TaskRuns do`

Reply:
Nice catch, thank you!